src/styles.ts

```diff
@@ -136,4 +136,39 @@ const widgetIframeStyle = {
   },
 }
 
-export { widgetBubbleStyle, widgetIframeStyle }
+const redirectionOverlayStyle: { [k: string]: Partial<CSSStyleDeclaration> } = {
+  overlay: {
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    top: '0',
+    left: '0',
+    background: 'rgba(10, 10, 10)',
+    position: 'fixed',
+    zIndex: '100001',
```

Comment:
why 100001? for z-index?

Reply:
its 1 greater than the widget zIndex so as to hide it.

Reply:
ok